zipline/protocol.py

```diff
@@ -59,6 +61,10 @@
     'share_count',
 ]
 
+TRADING_DAYS_PER_YEAR = 252
```

Comment:
These constants are specific to the NYSE calendar.  I don't think we should be enforcing them on all Zipline algorithms.